packages/kuma-gui/src/app/control-planes/components/ControlPlaneStatus.vue

```diff
@@ -5,133 +5,141 @@
-        v-if="props.canUseZones"
-        :total="props.globalInsight.zones.controlPlanes.total"
-        data-testid="zone-control-planes-status"
-      >
-        <template #icon>
-          <img
-            class="icon"
-            src="@/assets/images/zone.svg?url"
-            alt=""
-          >
-        </template>
-
-        <template #title>
-          {{ t('main-overview.detail.about.zone_control_planes') }}
-        </template>
-      </ResourceStatus>
-
-      <ResourceStatus
-        :total="props.globalInsight.meshes.total"
-        data-testid="meshes-status"
-      >
-        <template #icon>
-          <img
-            class="icon"
-            src="@/assets/images/mesh.svg?url"
-            alt=""
-          >
-        </template>
-
-        <template #title>
-          {{ t('main-overview.detail.about.meshes') }}
-        </template>
-      </ResourceStatus>
-
-      <template
-        v-for="meshServicesGeneric in [(['MeshService', 'MeshMultiZoneService', 'MeshExternalService'] as const).reduce((acc, curr) => ({ total: acc.total + props.globalInsight.resources[curr].total }), { total: 0 })]"
-        :key="typeof meshServicesGeneric"
+      <XLayout
+        v-if="props.globalInsight && !(props.globalInsight instanceof Error)"
+        type="columns"
+        class="columns-with-borders"
       >
         <ResourceStatus
-          :total="meshServicesGeneric.total || props.globalInsight.services.internal.total"
-          data-testid="services-status"
+          v-if="props.canUseZones"
+          :total="props.globalInsight.zones.controlPlanes.total"
+          data-testid="zone-control-planes-status"
         >
           <template #icon>
             <img
               class="icon"
-              src="@/assets/images/icon-wifi-tethering.svg?url"
+              src="@/assets/images/zone.svg?url"
               alt=""
             >
           </template>
 
           <template #title>
-            <XI18n
-              path="main-overview.detail.about.services"
-            />
+            {{ t('main-overview.detail.about.zone_control_planes') }}
           </template>
+        </ResourceStatus>
 
-          <template
-            v-if="meshServicesGeneric.total && props.globalInsight.services.internal.total > 0"
-            #description
-          >
-            <XI18n
-              path="main-overview.detail.about.descriptions.mesh_services"
-            />
+        <ResourceStatus
+          :total="props.globalInsight.meshes.total"
+          data-testid="meshes-status"
+        >
+          <template #icon>
+            <img
+              class="icon"
+              src="@/assets/images/mesh.svg?url"
+              alt=""
+            >
           </template>
 
+          <template #title>
+            {{ t('main-overview.detail.about.meshes') }}
+          </template>
+        </ResourceStatus>
+        <template
+          v-for="insight in [props.globalInsight]"
+          :key="typeof insight"
+        >
```

Comment:
I had to add this so that Vue understands that globalInsight is now guaranteed to be set and not an error, previously for some reason Vue couldn't figure that out from within the reduce statement.